src/gui/dialogs/campaign_selection.cpp

```diff
@@ -228,11 +229,31 @@ void campaign_selection::sort_campaigns(campaign_selection::CAMPAIGN_ORDER order
 		}
 	}
 
+	boost::dynamic_bitset<> vals = find_widget<multimenu_button>(this, "filter_completion", false).get_toggle_states();
```

Comment:
I think calling this `options` and/or giving each option a sensible variable name would be more readable.

Reply:
Updated variable name and added some comments, see if that works.